README.md

```diff
@@ -1,3 +1,5 @@
+> **Note**: We do not support running out of the git main branch. This is where development happens. Please use released versions via the published repo or git tags
```

Comment:
Made this explicit. Don't want users accidentally using main, as its a work in progress and could be broken.